tests/test_units/test_dataio.py

```diff
@@ -205,17 +207,13 @@ def test_content_invalid_string(globalconfig1):
 def test_content_invalid_dict(globalconfig1):
     with pytest.raises(ValidationError):
         ExportData(
-            config=globalconfig1, content={"not_valid": {"some_key": "some_value"}}
+            config=globalconfig1,
+            content={
+                "not_valid": {"some_key": "some_value"},
+            },
         )
 
 
-def test_content_valid_string(regsurf, globalconfig2):
```

Comment:
Given: https://github.com/equinor/fmu-dataio/pull/477/files#r1492624656 then this tests is obv. wrong.